packages/react/src/components/Flow/index.tsx

```diff
@@ -102,7 +102,7 @@ export function Flow({
   // }
 
   if (shouldForceMount || (!flow.isCompleted && !flow.isSkipped)) {
-    step.start()
+    step?.start()
```

Comment:
Note that this could be the beginning of a very long thread to pull: An entirely custom flow doesn't necessarily have to meet any of the assumptions we make internally about a flow having steps, etc.

For now, I don't think it's worth diving into, but going forward we may want to gracefully fail in codepaths that assume a standard flow structure.

Reply:
Totally. This null check is a good start!